src/backend/distributed/commands/view.c

```diff
@@ -457,16 +466,16 @@ PreprocessAlterViewStmt(Node *node, const char *queryString, ProcessUtilityConte
 	QualifyTreeNode((Node *) stmt);
 
 	EnsureCoordinator();
-	EnsureSequentialMode(OBJECT_VIEW);
 
 	/* reconstruct alter statement in a portable fashion */
 	const char *alterViewStmtSql = DeparseTreeNode((Node *) stmt);
 
-	List *commands = list_make3(DISABLE_DDL_PROPAGATION,
-								(void *) alterViewStmtSql,
-								ENABLE_DDL_PROPAGATION);
+	DDLJob *ddlJob = palloc0(sizeof(DDLJob));
+	ddlJob->targetObjectAddress = viewAddress;
+	ddlJob->metadataSyncCommand = alterViewStmtSql;
+	ddlJob->taskList = NIL;
 
-	return NodeDDLTaskList(NON_COORDINATOR_NODES, commands);
+	return list_make1(ddlJob);
```

Comment:
I've updated all ALTER VIEW process function to use metadata connection. I couldn't update DROP VIEW as we can not use metadata connection for drop commands, since we check the relation type while executing the command and we can not get the type for dropped objects after running them locally. I've also added test to show it.